### PR TITLE
PR to fix #342

### LINKED ISSUE #342: image normalization should be wrt the volume image - facilitates correctly displayed image colormap range
The way we normalize the image before saving to disk,

image

is relative to each image, which causes the following

image

labels are changing, but images are "mapped" to the same color! This might be only relevant to data, in which all pixel values in each segmented image are taking two unique values, but overall in the volume we have more unique values for the pixels.

Hence, the normalization should be wrt a global range in the volume of data.

### PR DESCRIPTION
**Pull request by fazamani**

### CODE REVIEW

File: experiments/interpretation/dutchf3_patch/local/test.py [hunk 1]
```diff
@@ -234,14 +236,14 @@ def _patch_label_2d(
                 path_prefix = f"{outdir}/{batch_indexes[i][0]}_{batch_indexes[i][1]}"
                 model_output = model_output.detach().cpu()
                 # save image:
-                image_to_disk(np.array(batch[i, 0, :, :]), path_prefix + "_img.png")
+                image_to_disk(np.array(batch[i, 0, :, :]), path_prefix + "_img.png", float(img.min()), float(img.max()))
```
Comment (yalaudah) [inline experiments/interpretation/dutchf3_patch/local/test.py:239]: Is it possible to use the max and min values here from the config file?

Comment (fazamani) [reply experiments/interpretation/dutchf3_patch/local/test.py:239]: do you mean now, or in the next attempt to address the "global MIN and MAX" next sprint? Currently we don't have MIN and MAX defined in the cofing, I reverted those changes!

Comment (yalaudah) [reply experiments/interpretation/dutchf3_patch/local/test.py:239]: okay that makes sense.. sorry if I misunderstood.

File: experiments/interpretation/dutchf3_patch/local/configs/hrnet.yaml [hunk 2]
```diff
@@ -16,6 +16,8 @@ DATASET:
   NUM_CLASSES: 6
   ROOT: "/home/username/data/dutch/data"
   CLASS_WEIGHTS: [0.7151, 0.8811, 0.5156, 0.9346, 0.9683, 0.9852]
+  MIN: 0
+  MAX: 7
```
Comment (yalaudah) [inline experiments/interpretation/dutchf3_patch/local/configs/hrnet.yaml:20]: are these values correct? shouldn't they be -1 to 1 or something like this?

File: experiments/interpretation/dutchf3_patch/local/configs/seresnet_unet.yaml [hunk 3]
```diff
@@ -15,6 +15,8 @@ DATASET:
   NUM_CLASSES: 6
   ROOT: /home/username/data/dutch/data
   CLASS_WEIGHTS: [0.7151, 0.8811, 0.5156, 0.9346, 0.9683, 0.9852]
+  MIN: 0
```
Comment (yalaudah) [inline experiments/interpretation/dutchf3_patch/local/configs/seresnet_unet.yaml:18]: same as above

File: experiments/interpretation/dutchf3_patch/local/configs/unet.yaml [hunk 4]
```diff
@@ -17,6 +17,8 @@ DATASET:
   NUM_CLASSES: 6
   ROOT: '/home/username/data/dutch/data'
   CLASS_WEIGHTS: [0.7151, 0.8811, 0.5156, 0.9346, 0.9683, 0.9852]
+  MIN: 0
+  MAX: 7
```
Comment (yalaudah) [inline experiments/interpretation/dutchf3_patch/local/configs/unet.yaml:21]: same as above

File: interpretation/deepseismic_interpretation/dutchf3/data.py [hunk 5]
```diff
@@ -549,13 +546,10 @@ class TestPatchLoader(PatchLoader):
     """
 
     def __init__(
-        self, data_dir, n_classes, stride=30, patch_size=99, is_transform=True, augmentations=None, debug=False
-    ):
+        self, config, is_transform=True, augmentations=None, debug=False
+    ): #stride=30, patch_size=99
```
Comment (yalaudah) [inline interpretation/deepseismic_interpretation/dutchf3/data.py:550]: I suggest we delete code we don't need instead of having it commented out.

File: interpretation/deepseismic_interpretation/dutchf3/data.py [hunk 6]
```diff
@@ -584,22 +578,20 @@ class TrainPatchLoader(PatchLoader):
...
         debug=False,
     ):
         super(TrainPatchLoader, self).__init__(
-            data_dir,
-            n_classes,
-            stride=stride,
-            patch_size=patch_size,
+            config,
+            # stride=stride,
+            # patch_size=patch_size,
```
Comment (yalaudah) [inline interpretation/deepseismic_interpretation/dutchf3/data.py:594]: Same comment as above: 
I suggest we delete code we don't need instead of having it commented out.
